modules/ort/src/main/kotlin/workflow/analyzers/OrtResultAnalyzer.kt

```diff
@@ -37,6 +40,11 @@ class OrtResultAnalyzer : ManualAnalyzer() {
 
     override fun getName() = "OrtResult"
 
+    override fun configure(configMap: Map<String, String>?) {
```

Comment:
I still believe we can safely make this a non-nullable type (as I already did [here](https://github.com/eclipse/antenna/blob/df5d05ad192e19415dc6e00a99c940439ae1b784/modules/ort/src/main/kotlin/workflow/processors/enricher/OrtDownloaderProcessor.kt#L30); the fix to also make the map immutable is [here](https://github.com/eclipse/antenna/pull/554)) as in all places I've seen so far an empty map is passed instead of `null`.